addons/cargo/functions/fnc_addCargoVehiclesActions.sqf

```diff
@@ -0,0 +1,42 @@
+/*
+ * Author: Dystopian
+ * Create actions for nearest vehicles with cargo.
+ *
+ * Arguments:
+ * 0: Target <OBJECT>
+ * 1: Player <OBJECT>
+ *
+ * Return Value:
+ * Children actions <ARRAY>
+ *
+ * Example:
+ * [target, player] call ace_cargo_fnc_addCargoVehiclesActions
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_target", "_player"];
+private ["_statement", "_actions", "_name", "_ownerName", "_icon", "_action"];
```

Comment:
please use `private keyword`. The variables from L31 onwards aren't even in the same script instance!